examples/wscript

```diff
@@ -1,6 +1,6 @@
 ## -*- Mode: python; py-indent-offset: 4; indent-tabs-mode: nil; coding: utf-8; -*-
 
 def build(bld):
-    obj = bld.create_ns3_program('woss-aloha-example', ['internet', 'mobility', 'stats', 'applications', 'uan', 'woss-ns3'])
+    obj = bld.create_ns3_program('woss-aloha-example', ['netanim', 'internet', 'mobility', 'stats', 'applications', 'uan', 'woss-ns3'])
```

Comment:
the example is not using netAnim explicitly. 
Is this really required?

Reply:
In general, it is good to have netanim support for post-processing purposes. I had modified the example as well to build support for netanim to test it, but as you said this example is not using netAnim explicitly so I only push changes related to wscript to pre-build netAnim support for any user by default in the example. 
It is solely up to user to use netanim for their simulation or not.

Reply:
Ok good, thanks! 
I will merge this tomorrow.